Website/Packages/datax-pipeline/src/modules/flowDefinition/flowHelpers.js

```diff
@@ -471,10 +475,17 @@ export function convertFlowToConfig(flow) {
         },
         outputs: sinkers,
         outputTemplates: outputTemplates,
-        rules: convertFlowToConfigRules(rules)
+        rules: convertFlowToConfigRules(rules),
+        batchList: batchList,
+        version: getTimeStamp()
     };
 }
 
+function getTimeStamp() {
+    let timeStamp = (new Date()).toISOString().slice(0, 19).replace(/-/g, "").replace(/:/g, "").replace("T", "");
```

Comment:
what are you doing here for the timeStamp? Can you add a comment as its not clear what all the magic replace is doing. BTW, why are you not using the "Moment" widely used 3rd datetime library? It would help address a lot of the common date operations/formating/assignments/conversions.

Reply:
no I didn't add this. it was probably added by accident when I merged with master. I removed this.